charts/gluu-all-in-one/values.yaml

```diff
@@ -18,6 +18,23 @@ state: TX
 cnPersistenceType: sql
 # -- Salt. Used for encoding/decoding sensitive data. If omitted or set to empty string, the value will be self-generated. Otherwise, a 24 alphanumeric characters are allowed as its value.
 salt: ""
+serviceAccountName: default
+customAnnotations:
+  adminUiIngress: {}
+  authServerIngress: {}
+  certificate: {}
+  clusterRoleBinding: {}
+  configMap: {}
+  cronjob: {}
+  deployment: {}
+  destinationRule: {}
+  horizontalPodAutoscaler: {}
+  gateway: {}
+  ingress: {}
```

Comment:
Please review all  ingresses as per ingress additional annotations exist which means this is a duplicate. Same goes for the microservices chart